tests/test_dataset.py

```diff
@@ -12,6 +12,38 @@
 import audbcards
 
 
+@pytest.mark.parametrize(
+    "db",
+    [
+        "medium_db",
+    ],
+)
+def test_dataset_property_scope(tmpdir, db, request):
+    r"""Test visibility of properties in local and global scopes."""
+    db = request.getfixturevalue(db)
+
+    dataset_cache = audeer.mkdir(tmpdir, "cache")
+    dataset = audbcards.Dataset(
+        db.name,
+        pytest.VERSION,
+        cache_root=dataset_cache,
+    )
+
+    props = [x for x in dataset.properties().keys()]
+
+    # should not exist in local scope
+    for prop in props:
+        assert prop not in vars()
+
+    # should not exist in global scope either
+    for prop in props:
+        assert prop not in globals()
+
```

Comment:
I think would be nice to also test, that the properties are now part of `Dataset`, maybe:

```suggestion

for prop in props:
    assert hasattr(dataset, prop)
     
```
```

Reply:
This makes sense and also passes.

Reply:
Ah, sorry, I messed up the indention. Could you please fix the code and push the fix?

Reply:
Done. github mentions the commit below too: 4047ad